Learn-Information-Security-by-Building-a-Contact-Form/server050.js

```diff
@@ -0,0 +1,12 @@
+const express = require("express");
+const app = express();
+const helmet = require("helmet");
+// https://helmetjs.github.io/ - for more information on all packages and uses.
+
+app.use(helmet.dnsPrefetchControl());
+app.use(helmet.hidePoweredBy());
```

Comment:
Try to include a note about why `X-Powered-By` can be dangerous, then have learners call it with `app.use(helmet.hidePoweredBy());`